health/notifications/awssns/README.md

```diff
@@ -92,8 +92,6 @@ The supported variables are:
 | `${url_name}`               | Same as ${name} but URL encoded                                                  |
 | `${chart}`                  | The name of the chart (type.id)                                                  |
 | `${url_chart}`              | Same as ${chart} but URL encoded                                                 |
-| `${family}`                 | The family of the chart                                                          |
```

Comment:
@Ancairon these notification readmes are generated, right? If so, we shouldn't upate them manually @MrZammler

Reply:
Indeed it is, look at the note at the top of the file https://github.com/netdata/netdata/pull/16025/files#diff-8cc0362d13426093c886b89dd5b6f4465081c299062c81436df3c8696435ab41L7

Reply:
@MrZammler you need to edit the yaml file for the changes to take effect.

Reply:
Ah, didn't see that, sorry. I've also edited the yaml file as part of this PR, should I revert the README back, or it's not a problem?